panel/io/resources.py

```diff
@@ -258,7 +258,7 @@ def component_resource_path(component, attr, path):
     component_path = COMPONENT_PATH
     if state.rel_path:
         component_path = f"{state.rel_path}/{component_path}"
-    rel_path = os.fspath(resolve_custom_path(component, path, relative=True)).replace(os.path.sep, '/')
+    rel_path = str(resolve_custom_path(component, path, relative=True)).replace(os.path.sep, '/')
```

Comment:
Does it make sense to use `rel_path` if its output is `None`? Maybe we could do something like this. 
```suggestion
    resolve_path = resolve_custom_path(component, path, relative=True)
    rel_ath = os.fspath(resolve_path).).replace(os.path.sep, '/') if resolve_path else ""

```